src/adapters/CKEditorAdapter.ts

```diff
@@ -14,51 +14,28 @@
  * limitations under the License.
```

Comment:
Maybe this file should be name CKEditor4Adapter.ts

Reply:
@JustinHorn Changing the class name would break compatibility for existing CK4 adapter users.

example: Existing integrations using Ck4 adapter with no requirement to update to CK5 will get a compilation error on updating the SDK.

Reply:
okay 👍🏻